adds that farm your dad talked about sending your old pet catperson to so they could live out the rest of their lives free

MMMiracles opened the pull request:
:cl: MMMiracles
add: A brand new space-farm, where your family sends all your old/sick catpeople to live out the rest of their days being free to roam the acres and chase the field grayshirts.
/:cl:

[why]: It exists for lizards, so why not catpeople?

MMMiracles committed:
adds a space ruin for where dad tells you they sent your pet catperson to live on a farm where they can be free and chase all the grayshirts they want

Bawhoppen commented:
You're a pretty funny guy MMMiracles, ya know that?

MMMiracles commented:
thank you ill be here all week

ZeWaka commented:
I like this PR. I think it adds to the backstory and the overall lore of the catpeople. By introducing the concept of a hospice/elderly home into their lore, the potential for further development is expanded greatly. The lore behind the Animal Rights Consortium is lacking, and definitely could be expanded within this PR.
This PR and process can also drive both profit and comfort to the station through the use of PRs #28732 and #28395, and contribute positively on both ends of the equation.

Iamgoofball commented:
Good PR

Iamgoofball commented:
This PR has my full support.

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/item/mop,
+/obj/item/reagent_containers/glass/bucket,
+/obj/effect/decal/cleanable/cobweb{
+	icon_state = "cobweb2"
```
/obj/effect/decal/cleanable/cobweb/cobweb2

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"al" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
```
dirty var

Xhuis replied:
😩

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"al" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"am" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"am" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"an" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"an" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"ao" = (
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	tag = "icon-manifold (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ao" = (
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	tag = "icon-manifold (NORTH)";
+	icon_state = "manifold";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ap" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHWEST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"aw" = (
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"aw" = (
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	dir = 1
+	},
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	},
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/blood/old,
+/obj/machinery/light/small{
+	tag = "icon-bulb1 (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/blood/old,
+/obj/machinery/light/small{
+	tag = "icon-bulb1 (NORTH)";
+	icon_state = "bulb1";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/light/small{
+	tag = "icon-bulb1 (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/light/small{
+	tag = "icon-bulb1 (NORTH)";
+	icon_state = "bulb1";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"aM" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"aM" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"aO" = (
+/obj/machinery/door/firedoor/border_only{
+	tag = "icon-door_open (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"aO" = (
+/obj/machinery/door/firedoor/border_only{
+	tag = "icon-door_open (NORTH)";
+	icon_state = "door_open";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"aU" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/item/twohanded/required/kirbyplants/random,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/item/twohanded/required/kirbyplants/random,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"aW" = (
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"aW" = (
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTHEAST)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	name = "cat meat"
+	},
+/obj/machinery/light/small{
+	tag = "icon-bulb1 (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	},
+/obj/machinery/light/small{
+	tag = "icon-bulb1 (NORTH)";
+	icon_state = "bulb1";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bg" = (
+/turf/open/floor/plasteel/white/corner{
+	tag = "icon-whitecorner (WEST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bg" = (
+/turf/open/floor/plasteel/white/corner{
+	tag = "icon-whitecorner (WEST)";
+	icon_state = "whitecorner";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bh" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (EAST)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/vomit/old,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bm" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bm" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bn" = (
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	tag = "icon-manifold (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bn" = (
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	tag = "icon-manifold (EAST)";
+	icon_state = "manifold";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bp" = (
+/obj/structure/chair/wood/wings{
+	tag = "icon-wooden_chair_wings (WEST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bp" = (
+/obj/structure/chair/wood/wings{
+	tag = "icon-wooden_chair_wings (WEST)";
+	icon_state = "wooden_chair_wings";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bs" = (
+/mob/living/simple_animal/hostile/cat_butcherer,
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/mob/living/simple_animal/hostile/cat_butcherer,
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (EAST)";
+	icon_state = "whitehall";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/item/clothing/gloves/color/latex,
+/obj/item/surgical_drapes,
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (WEST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/item/surgical_drapes,
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (WEST)";
+	icon_state = "whitehall";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	pixel_x = 24
+	},
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	},
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (EAST)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bA" = (
+/turf/open/floor/plasteel/white/corner{
+	tag = "icon-whitecorner (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bA" = (
+/turf/open/floor/plasteel/white/corner{
+	tag = "icon-whitecorner (EAST)";
+	icon_state = "whitecorner";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bB" = (
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bB" = (
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (NORTH)";
+	icon_state = "whitehall";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bC" = (
+/turf/open/floor/plasteel/white/corner{
+	tag = "icon-whitecorner (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bC" = (
+/turf/open/floor/plasteel/white/corner{
+	tag = "icon-whitecorner (NORTH)";
+	icon_state = "whitecorner";
```
dirty var

MMMiracles commented:
ALRIGHT MAN I GET IT THERE'S A LOT OF TAGS

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bG" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (SOUTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (SOUTHEAST)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bJ" = (
+/obj/structure/closet/crate/bin,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/structure/closet/crate/bin,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bK" = (
+/obj/structure/table,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/structure/table,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bL" = (
+/obj/structure/chair,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/structure/chair,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bM" = (
+/obj/structure/reagent_dispensers/water_cooler,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/structure/reagent_dispensers/water_cooler,
+/turf/open/floor/plasteel/neutral/side{
+	tag = "icon-neutral (NORTH)";
+	icon_state = "neutral";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bR" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bR" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bS" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHWEST)";
```
dirty var

ShizCalev commented:
note to self; never use smartgit commenting for reviews again

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bS" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHWEST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bU" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	name = "cat meat"
+	},
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	},
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (SOUTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"bZ" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"bZ" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"ca" = (
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	tag = "icon-manifold (NORTH)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ca" = (
+/obj/machinery/atmospherics/pipe/manifold/green/hidden{
+	tag = "icon-manifold (NORTH)";
+	icon_state = "manifold";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"cc" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHWEST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"cc" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHWEST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"ce" = (
+/obj/machinery/atmospherics/pipe/simple/dark/hidden{
+	tag = "icon-intact (SOUTHWEST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ce" = (
+/obj/machinery/atmospherics/pipe/simple/dark/hidden{
+	tag = "icon-intact (SOUTHWEST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ch" = (
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer{
+	tag = "icon-freezer (EAST)";
+	icon_state = "freezer";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"ch" = (
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer{
+	tag = "icon-freezer (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ci" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/manifold/dark/hidden{
+	tag = "icon-manifold (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/manifold/dark/hidden{
+	tag = "icon-manifold (EAST)";
+	icon_state = "manifold";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"cj" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHEAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"cj" = (
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (NORTHEAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+"ck" = (
+/obj/effect/spawner/structure/window/reinforced,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/spawner/structure/window/reinforced,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
+	icon_state = "intact";
```
dirty var

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+	id = "meow";
+	name = "kitty protection door"
+	},
+/obj/machinery/door/window{
```
replace with door/window/westleft

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,2009 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/vomit/old,
+/obj/machinery/atmospherics/pipe/simple/green/hidden{
+	tag = "icon-intact (EAST)";
```
dv

81Denton commented on code/modules/mob/living/simple_animal/hostile/cat_butcher.dm:
```diff
@@ -0,0 +1,43 @@
+	if(. && prob(35) && iscarbon(target))
+		var/mob/living/carbon/human/L = target
+		if(L.dna.features["tail_human"] == "Cat")
+			visible_message("[src] severs [L]'s tail in one swift swipe!", "<span class='notice'>You sever [L]'s tail in one swift swipe.</span>")
```
That's amazing, good PR.

arsserpentarium commented:
Lovely to see.
Lizard death camp is equipped with gas delivery system. And torture room is hidden in backroom.
Catperson death camp without gas chambers,but allow to slowly dissect one victim before eyes of all other prisoners. Lovely.
We need ghost antag role,which start in such facility. And must kidnap and dissect given race to put their guts into mining-like vendor to get more cool toys.
Like abductor,but just maniac.

Bawhoppen commented:
The real reason this PR is so great, is that it will solidify catbeasts as having a permanent place in tg-station code. Nobody can try to remove them after this...

arsserpentarium commented:
That's why you should be very careful with genocide:
You can try very hard. But if you will fail, or use wrong media policy, you'll only enforce their positions.

ma44 commented:
A ghost role spawner that it's only purpose is to hunt down people with tails, harvest and probably also have some form of restraints and actively hunt out the station can't possibly go wrong at all.

Buggy123 commented:
I wondered why there were so many comments on this, I opened it expecting a raging discussion. Instead I saw only a dozen or so... and nearly a hundred "dirty var" reviews from Shiz.

MMMiracles committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into cats_genetics_questionmark

# Conflicts:
#	code/datums/ruins/space.dm

MMMiracles committed:
dirty vars

ShizCalev committed:
Vars

MMMiracles committed:
Merge pull request #18 from ShizCalev/cats_genetics_questionmark

Vars

MMMiracles committed:
changes windoor to proper subtype, removed extra circular saw

MMMiracles committed:
Merge branch 'cats_genetics_questionmark' of https://github.com/MMMiracles/-tg-station into cats_genetics_questionmark

MMMiracles commented:
c'mon guys the cats need their home

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,1913 @@
+/area/template_noop)
+"ab" = (
+/turf/closed/mineral,
+/area/space)
```
/area/ruin/unpowered

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,1913 @@
+/area/space)
+"ac" = (
+/turf/open/floor/plating/asteroid/airless,
+/area/space)
```
/area/ruin/unpowered

ShizCalev reviewed (changes requested):
last ones.

MMMiracles committed:
area/space to area/ruin/unpowered, added gps tag

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,1919 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"aN" = (
+/turf/closed/mineral,
+/area/space)
```
/area/ruin/unpowered/no_grav

ShizCalev reviewed (changes requested):
you added another :(

MMMiracles committed:
in area/space nobody can hear you scream

ShizCalev commented on _maps/RandomRuins/SpaceRuins/mrow_thats_right.dmm:
```diff
@@ -0,0 +1,1916 @@
+/area/ruin/space/has_grav/powered/cat_man)
+"ch" = (
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer{
+	icon_state = "freezer";
```
dv

MMMiracles committed:
Update mrow_thats_right.dmm

if i scream, scream, scream, about a good man's life, would you ever stop and listen, would you open up your eyes

ShizCalev commented:
@MMMiracles conflict.

MMMiracles committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into cats_genetics_questionmark

# Conflicts:
#	icons/mob/simple_human.dmi

MMMiracles committed:
cat

MMMiracles committed:
Merge branch 'cats_genetics_questionmark' of https://github.com/MMMiracles/-tg-station into cats_genetics_questionmark

MMMiracles commented:
ok lessa go

WJohn reviewed (commented):
Sprites are fine.
